### DISCUSSION TITLE
Pages not working?

### DISCUSSION THREAD

**Discussion by Pixelest** [xenocrat/chyrp-lite#182] [Q&A] [answered]
So I am trying to add pages to my blog. I created 2 test pages to see where they would display but I am not seeing any change to my home page. Is there supposed to be links created to these new pages? 
![image_2023-01-16_141157817](https://user-images.githubusercontent.com/2861198/212751121-3aed0a06-5af4-432e-953b-e2b86eecfce9.png)
![image](https://user-images.githubusercontent.com/2861198/212751150-c6862b59-c50c-4ba0-bc9b-0c3101ddc6d7.png)

**Comment by xenocrat** [comment, marked as answer]
Umbra doesn’t list pages. Pages are fully supported by the theme, but not listed in the navigation. Take a look [here](https://github.com/xenocrat/chyrp-lite/blob/master/themes/blossom/content/sidebar.twig#L61) to see how Blossom lists pages in the sidebar - you could modify Umbra to do the same in the top navigation.
